modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/TypeUtils.java

```diff
@@ -704,7 +706,7 @@ private static TypeSpec convertToTypeSpec(RelDataType type) {
         }
     }
 
-    /** Check limitation for character types and throws exception if row contains character sequence more than type defined.
+    /** Check limitation for character and binary types and throws exception if row does not fit into type defined
```

Comment:
please return back period at the end of the sentence

Reply:
done